src/connector/src/parser/mod.rs

```diff
@@ -579,14 +583,14 @@ async fn into_chunk_stream<P: ByteStreamSourceParser>(mut parser: P, data_stream
                     match txn_ctl {
                         TransactionControl::Begin { id } => {
                             if let Some(Transaction { id: current_id, .. }) = &current_transaction {
-                                tracing::warn!(parent: &parse_span, current_id, id, "already in transaction");
+                                tracing::warn!(current_id, id, "already in transaction");
```

Comment:
Transaction is only applicable to CDC source, where `split_id` and `offset` is not that useful. So simply omit them here.